lib/AFS/CellCC/Config.pm

```diff
@@ -107,6 +107,8 @@ my @directives = (
     { key => 'restore/check-interval', default => 60, },
     { key => 'restore/queues', type => 'HASH', default => {}, },
     { key => qr:^restore/queues/[^/]+/max-parallel$:, default => 1, },
+    { key => qr:^restore/queues/[^/]+/release/flags$:, type => 'HASH', default => {}, },
+    { key => qr:^restore/queues/[^/]+/release/flags/+:, default => undef, },
```

Comment:
is this line needed? I would think this would be handled by the previous line (or the previous line is unnecessary), but I may not be remembering the config parsing/checking right. Do we actually need both?

if we do need this, that regex I think should be qr:^restore/queues/[^/]+/release/flags/[^/]+$:

Reply:
I think we need both. Without line 111, we get messages like (from the check-server):

Configuration error: Unknown config directive 'restore/queues/<queue>/release/flags/<flag>'.

Line 110 seems to be unnecessary. However, if we do not have an entry for the default queue (restore/queues/default/release/flags), config_get (Restore.pm:475) will fail. With line 110, if we do not have an entry for the default queue, we get an empty hash (and config_get does not fail).

> if we do need this, that regex I think should be
> qr:^restore/queues/[^/]+/release/flags/[^/]+$:

Agreed.